README.md

```diff
@@ -0,0 +1,191 @@
+    $ cd sti-ruby
+    $ make test VERSION=2.0
+    ```
+
+**Notice: By omitting the `VERSION` parameter, the build/test action will be performed
+on all the supported versions of Ruby. Since we are now supporting only version `2.0`
+you can omit this parameter.**
+
+
+Repository organization
+------------------------
+* **`<ruby-version>`**
+
+    * **Dockerfile**
+
+        CentOS based Dockerfile.
+
+    * **Dockerfile.rhel7**
+
+        RHEL based Dockerfile. In order to perform build or test actions on this
+        Dockerfile you need to run the action on properly subscribed RHEL machine.
+
+    * **`.sti/bin/`**
+
+        This folder contains scripts that are run by [STI](https://github.com/openshift/source-to-image):
+
+        *   **assemble**
+
+            Is used to install the sources into location from where the application
+            will be run and prepare the application for deployment (eg. installing
+            modules using npm, etc..)
+
+            Following [markers](#markers) take effect in `assemble` script:
+            - disable_asset_compilation
+
+        *   **run**
+
+            This script is responsible for running the application, by using the
+            application web server.
+
+    * **`ruby/`**
+
+        This folder contains file with commonly used modules.
+
+    * **`test/`**
+
+        This folder is containing [STI](https://github.com/openshift/source-to-image)
+        test framework with simple Rack server.
+
+        * **`test-app/`**
+
+            Simple Rack server used for testing purposes in the [STI](https://github.com/openshift/source-to-image) test framework.
+
+        * **run**
+
+            Script that runs the [STI](https://github.com/openshift/source-to-image) test framework.
+
+* **`hack/`**
+
+    Folder contains scripts which are responsible for build and test actions performed by the `Makefile`.
+
+Image name structure
+------------------------
+##### Structure: openshift/1-2-3
+
+1. Platform name - ruby
+2. Platform version(without dots)
+3. Base builder image - centos7/rhel7
+
+Examples: `openshift/ruby-20-centos7`, `openshift/ruby-20-rhel7`
+
+
+Environment variables
+---------------------
+
+*  **RACK_ENV**
+
+    This variable specifies in what environment should the ruby application be deployed - `production`, `development`, `test`.
+    Each level has different behavior in terms of logging verbosity, error pages, ruby gem installation, etc.
+
+    **Note**: The application assets are going to be compiled only if the `RACK_ENV` is set to `production`
+
+*  **RAILS_ENV**
+
+    This variable specifies in what environment should the ruby application be deployed - `production`, `development`, `test`.
+    Each level has different behavior in terms of logging verbosity, error pages, ruby gem installation, etc.
+
+    **Note**: The application assets are going to be compiled only if the `RAILS_ENV` is set to `production`
+
+* **STI_LOCATION** (default: `/usr/local/sti`)
```

Comment:
I think we are not putting this into sti-base and we are not using this var anymore, @soltysh am I hallucinating? 

if we put this into sti-base, it should be documented there. Reserve this documentation ONLY for ruby/language specific things.